cms/djangoapps/contentstore/views/item.py

```diff
@@ -146,9 +147,14 @@ def xblock_handler(request, usage_key_string):
 
             if 'application/json' in accept_header:
                 fields = request.GET.get('fields', '').split(',')
+                info_type = request.GET.get('info_type')
                 if 'graderType' in fields:
                     # right now can't combine output of this w/ output of _get_module_info, but worthy goal
                     return JsonResponse(CourseGradingModel.get_section_grader_type(usage_key))
+                elif info_type == 'ancestor':
```

Comment:
Nit: I wonder if you should pass this in fields in the same way as graderType above, rather than introducing a new info_type parameter.